src/Symfony/Component/Console/Question/Question.php

```diff
@@ -85,6 +86,27 @@ public function setMultiline(bool $multiline): static
         return $this;
     }
 
+    /**
+     * Returns the timeout in seconds.
+     */
+    public function getTimeoutSeconds(): ?int
+    {
+        return $this->timeoutSeconds;
+    }
+
+    /**
+     * The timeout is the maximum time the user has to answer the question.
+     * If the user does not answer within this time, an exception will be thrown.
+     *
+     * @return $this
+     */
+    public function setTimeout(?int $seconds): static
```

Comment:
The method name is inconsistent with the getter. I would recommend to use `getTimeout()` and `setTimeout()` as method names to be consistent with othe places in the framework (and so also rename the property to `timeout`).

Reply:
Adjusted.